internal/accounts/create.go

```diff
@@ -134,3 +147,301 @@ func create(
 		include: createFlags.Include,
 	}, nil
 }
+
+func createInteractive(state *flowkit.State, loader flowkit.ReaderWriter) (*flow.Account, error) {
+	log := output.NewStdoutLogger(output.InfoLog)
+
+	log.Info(fmt.Sprintf("%s What name would you like to give this new account?", output.GoEmoji()))
```

Comment:
This question should be part of the prompt. We should avoid emoji here, so we don't overload the users screen with emjois, they are meant to be used when we really need to get users attention to something important, like warning, or confirmation of action etc.

Reply:
addressed